src/gov/nist/javax/sip/stack/UDPMessageChannel.java

```diff
@@ -318,6 +318,9 @@ public void run() {
 
```

Comment:
you could also modify the catch exception to catch throwable and be done with it. 1 line fix.

Reply:
done!